test/cmdlineTests/ast_function_location/input.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.17;
+
+function checkPattern(int _x, int _y,int _xStep, int _yStep)  returns(int startX, int startY, int endX, int endY)
+{ 
```

Comment:
```suggestion
{
```

Please configure your editor to remove trailing whitespace automatically. The [`chk_coding_style`](https://app.circleci.com/pipelines/github/ethereum/solidity/28451/workflows/abf89681-cc6c-4ba0-97b7-f32e9e8ce9f0/jobs/1263155) job is failing because of this.